packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -450,42 +450,47 @@ export default class TableColumnResizeEditing extends Plugin {
 	 * @private
 	 */
 	_setupColumnResizers() {
-		const editor = this.editor;
-		const editingView = editor.editing.view;
-
-		editingView.addObserver( MouseEventsObserver );
-		editingView.document.on( 'mousedown', this._onMouseDownHandler.bind( this ), { priority: 'high' } );
-
 		const domEmitter = Object.create( DomEmitterMixin );
 
+		domEmitter.listenTo( global.window.document, 'mousedown', this._onMouseDownHandler.bind( this ), { priority: 'high' } );
 		domEmitter.listenTo( global.window.document, 'mouseup', this._onMouseUpHandler.bind( this ) );
+		domEmitter.listenTo( global.window.document, 'drop', this._onMouseUpHandler.bind( this ), { useCapture: true } );
```

Comment:
Why do we use drop listener? I believe that dropping something should also trigger `mouseup` event. Unless... we're cancelling it in our drag and drop handler 🙂

Either way we should shortly explain this in a comment.

Reply:
While working on this issue, the thing with drag&drop got complicated and cannot be handled here anymore. So I decided to revert this change and work on it in other PR as we have a dedicated ticket for this: #11772.